sdk/appcenter/src/test/java/com/microsoft/appcenter/AbstractAppCenterTest.java

```diff
@@ -105,6 +105,12 @@ static void addArgumentToRegistry(String value) {
         when(InstrumentationRegistryHelper.getArguments()).thenReturn(mockBundle);
     }
 
+    static void addRunningInAppCenterToRegistry(String value) {
```

Comment:
Could move this method and all the new tests to a new dedicated sub-class.
Also there are missing tests that make code coverage check fail.